From 77631e55a93266b528ad596cfd90454b9856ed81 Mon Sep 17 00:00:00 2001
From: MithicSpirit <rpc01234@gmail.com>
Date: Sat, 26 Feb 2022 12:26:08 -0500
Subject: [PATCH 2/3] batdiff: Fix use of deprecated --hunk-style

Delta has deprecated --hunk-style in favor of its synonym
--hunk-header-decoration-style. See
https://github.com/dandavison/delta/releases/tag/0.12.0
---
 src/batdiff.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/batdiff.sh b/src/batdiff.sh
index f1c7e1c..2cc8d1e 100755
--- a/src/batdiff.sh
+++ b/src/batdiff.sh
@@ -77,7 +77,7 @@ done
 
 # Append arguments for delta/bat.
 BAT_ARGS+=("--terminal-width=${OPT_TERMINAL_WIDTH}" "--paging=never")
-DELTA_ARGS+=("--width=${OPT_TERMINAL_WIDTH}" "--paging=never" "--hunk-style=plain")
+DELTA_ARGS+=("--width=${OPT_TERMINAL_WIDTH}" "--paging=never" "--hunk-header-decoration-style=plain")
 
 if "$OPT_COLOR"; then
 	BAT_ARGS+=("--color=always")
-- 
2.36.0.44.g0f828332d5

